test/integration/setting_js_test.rb

```diff
@@ -29,4 +29,78 @@ class SettingJSTest < IntegrationTestWithJavascript
     assert_index_page(settings_path, "Settings", false, true, false)
     assert page.has_link?("My Pretty Setting Label", :href => "#category_label_test_settings_tab")
   end
+
+  test "general headers values" do
+    visit settings_path
+
+    expect page.has_content?('Name')
+    expect page.has_content?('Value')
+    expect page.has_content?('Description')
+  end
+
+  test "edit single arrays values" do
+    visit settings_path
+    find('button#http_proxy_except_list').click
```

Comment:
You can use the [`click_button([locator])`](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Actions#click_button-instance_method) method. Quoting that:

> Finds a button on the page and clicks it. This can be any <input> element of type submit, reset, image, button or it can be a <button> element. All buttons can be found by their id, name, [test_id](https://www.rubydoc.info/github/jnicklas/capybara/master/Capybara#configure-class_method) attribute, value, or title. <button> elements can also be found by their text content, and image <input> elements by their alt attribute.

Looks like we can set `test_id` somewhere in our setup code to `data-ouia-component-id` as well so you don't need to think about that too much in the code.